EIPS/eip-7756.md

```diff
@@ -58,13 +58,13 @@ Each object contains the following members.
 | `op`         | Number     | Yes         | Opcode                                              |
 | `immediate`  | Hex-String | *see below* | Immediate argument of the Opcode                    |
 | `gas`        | Hex-Number | Yes         | Gas left before executing this operation            |
-| `gasCost`    | Hex-Number | No          | Gas cost of this operation                          |
+| `gasCost`    | Hex-Number | Yes         | Gas cost of this operation                          |
 | `memory`     | Array      | No          | Array of all allocated values                       |
 | `memSize`    | Number     | Yes         | Size of memory array                                |
 | `stack`      | Array      | Yes         | Array of all values on the stack                    |
-| `depth`      | Number     | Yes         | Depth of the call stack                             |
-| `fdepth`     | Number     | Yes if EOF  | Depth of the [EIP-4750](./eip-4750.md) return stack |
-| `returnData` | Hex-String | No          | Data returned by function call                      |
+| `depth`      | Number     | Yes         | Depth of the contract call stack (non-CALLF)        |
+| `functionDepth`     | Number     | Yes if EOF  | Depth of the [EIP-4750](./eip-4750.md) CALLF return stack |
```

Comment:
`fDepth` is changed to `functionDepth` here, however, there 2 `fdepth` still in the file (which should thus be renamed).

Reply:
fixed.